src/js/view/view.js

```diff
@@ -250,7 +246,24 @@ define([
                     });
                 }
             }
-            return bounds;
+        }
+
+        function _responsiveListener() {
+            if (!document.body.contains(_playerElement)) {
+                window.removeEventListener('resize', _responsiveListener);
+                if (_isMobile) {
+                    window.removeEventListener('orientationchange', _responsiveListener);
+                }
+            } else {
+                if (window.requestAnimationFrame) {
```

Comment:
Looks like this is not supported in IE9, will that be an issue?

https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame#Browser_compatibility